src/agent/internals/src/main/java/com/google/devtools/cdbg/debuglets/java/UniquifierComputer.java

```diff
@@ -58,9 +57,18 @@ public UniquifierComputer(String initializationVector, SortedSet<String> applica
     }
   }
 
+  private static final char[] HEX_DIGITS = "0123456789ABCDEF".toCharArray();
+
   /** Computes the SHA1 hash value and encodes it in a string. */
   public String getUniquifier() {
-    return DatatypeConverter.printHexBinary(hash.digest());
+    byte[] digest = hash.digest();
+    char[] hex = new char[digest.length * 2];
+    for (int i = 0, j = 0; i < digest.length; i++) {
+      byte b = digest[i];
+      hex[j++] = HEX_DIGITS[(b >> 4) & 0xf];
+      hex[j++] = HEX_DIGITS[b & 0xf];
+    }
+    return new String(hex);
```

Comment:
I would love to have a teeny tiny unit test for the byte[] to char[] conversion.

Reply:
Done.